### ISSUE TITLE
There has been some blank lines

### ISSUE THREAD

**Issue by CraneHuang** [clojure-emacs/ac-cider#28]
Sometimes,some blank lines appear after the code completed.as you can see in the following picture:
![2015-03-19 08 56 29](https://cloud.githubusercontent.com/assets/5162826/6722565/0b2f6d78-ce17-11e4-9329-02e3dbf38493.png)

**Comment by alexander-yakushev** [comment]
Does it happen only when you complete from REPL, or in Clojure buffers too?

Is it consistent for specific completions you make, or seems random?

**Comment by CraneHuang** [comment]
Both in REPL and Clojure buffers.And it seems random.

**Comment by alexander-yakushev** [comment]
I suspect it might be a bug in popul.el, because ac-cider doesn't interact with the buffer directly anywhere. But if you happen to spot some pattern when it happens, I will investigate.

**Comment by CraneHuang** [comment]
Ok,if I find some thing related , I'll tell you here.

**Comment by asadeghi** [comment]
This happens to me all the time. To repro reliably:
1. Turn on "whitespace-mode" so you can see the extra blank lines.
2. Edit something with auto-complete in a Clojure buffer that is towards the end of the file, i.e. the last few lines. As the auto complete kicks in you can see the extra lines being added (which is expected).
3. When the auto-complete window is showing you options, wait 3-5 seconds, then press enter or type something new. This will cause the issue in question. 
Alternatively - with the AC window showing, press ESC.

**Comment by alexander-yakushev** [comment]
Yes, I've been aware of that for some time. Unfortunately it is a bug in
auto-complete itself - this happens to me on ac-slime for common lisp as
well.

You might try to submit a report to AC repo and see if anyone reacts.

**Comment by MalloZup** [comment]
autogenerated with https://github.com/MalloZup/doghub:  issue inactive since 450 days. Please update the issue or close it
